js/bars.js

```diff
@@ -150,15 +150,16 @@ historian.Bars = function(context, barData, levelData, timeToDelta, state,
     }.bind(this));
 
     this.container_.find(REGEXP_FILTER_).on('input',
-        function(event) {
-          var input = $(event.target);
-          var isValid = historian.utils.isValidRegExp(
-              /** @type {string} */ (input.val()));
-          input.toggleClass('error', !isValid);
-          if (isValid) {
-            this.updateSeries_();
-          }
-        }.bind(this));
+      function(event) {
+        var input = $(event.target);
+        var isValid = historian.utils.isValidRegExp(
+            /** @type {string} */ (input.val()));
+        input.toggleClass('error', !isValid);
+        if (isValid) {
+          this.updateSeries_();
+        }
+      }
+    .bind(this));
```

Comment:
.bind is on the line wiith } in earlier part of the code.  see line 431. Please be consistent